dev-docs/RFCs/proposals/pydeck-interactive-api-rfc.md

```diff
@@ -32,21 +32,34 @@ Question: Should we use ipywidgets style API, or deck.gl API, or transport API o
 
 ## Use Cases
 
-### "Raw" Events (Click and Hover, etc)
+### Events (Click and Hover, etc)
 
-Hover
+**Hover Tooltip**
```

Comment:
Maybe worth noting: Implementation for this should likely go into the JSON API, since it's one-way. It should be done using the [deck.gl getTooltip parameter](https://github.com/visgl/deck.gl/blob/7b3b615edb86c7a6adf9230bdcb139a58e9933d3/docs/api-reference/deck.md#gettooltip-function-optional) within the JSON API but instead is done within the widget layer.